site/docs/components/autocomplete.mdx

```diff
@@ -0,0 +1,105 @@
+- **Categorization:** Consider if it's helpful to chunk results into categories with subheadings.
+- **Good defaults:** Where possible, show good defaults on focus, even before the user has typed anything.
+- **Keyboard navigation:** Ensure the results list can be navigated with a keyboard and test this regularly.
+
+### Matching results
+
+- **Fuzzy search**: a user can write some text and a partial match returns results. Consider whether the text needs to be in order, what effect spaces have, and the effect of capitalization on the match. For example, a forgiving search input that lets you type “design” to match the result “Marketing Design” may be ideal for users that aren't sure what exactly it will be called here.
+- **Smart capitalization matching**: consider appropriate capitalization matching for the data available. For example, when people type a search query using lowercase letters, you might return all, case-insensitive results and when they type a search query containing capital letters, you might return only case-sensitive results matching their capitalization.
+
+
+### Content
+
+Use short and clear keywords in autocomplete options. This helps people understand the content associated with the keyword. You might, for example, use linked page titles as keywords rather than page descriptions.
+
+### Autosuggest
+
+Similar to traditional “autocomplete,” “autosuggest” breaks beyond the input provided to suggest alternative, relevant answers. It might even suggest results from multiple data sets.
+
+### Accessibility
+
+See [WAI-ARIA Authoring Practices 1.1: Combo Box](https://www.w3.org/TR/wai-aria-practices-1.1/#combobox) and [ARIA 1.1 Combobox with Listbox Popup Examples](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html) for more details.
+
+#### Keyboard support
+
+- When tabbing forward or backward through the page, let the autocomplete text input take keyboard focus with the tab key.
+- Let people navigate through the list of autocomplete options using the up and down arrow keys.
+- Let people select the option that has focus using the Enter key.
+
+#### Announcements
+
+Let people using screen readers know when new options are presented while typing. This requires testing to avoid excessively noisy announcements.
+
+If the Autocomplete component causes a change to content on the page, use ARIA live regions to inform screen reader users of what's changed.
+
+## When to use and when not to use
+
+<WhenToUseAndWhenNotToUse>
+<WhenToUse>
+
+- Use Autocomplete when the user has some idea of the content available and some related terms they might use to search.
+
+</WhenToUse>
+<WhenNotToUse>
+
+- Avoid Autocomplete when the user has no concept of the search space and what's available. You might instead use navigation options or surface top content.
+
+</WhenNotToUse>
+</WhenToUseAndWhenNotToUse>
+
+## See also
+
```

Comment:
All these period characters following not-a-sentence list items feel a bit weird. Do we do that everywhere?

Reply:
CC @berry-thanki I'd be keen to have a clear stance on this and clean it up site wide